libs/hscim/src/Web/Scim/Schema/Error.hs

```diff
@@ -175,6 +181,16 @@ serverError details =
 ----------------------------------------------------------------------------
 -- Servant
 
+-- | Convert a SCIM 'Error' to a Servant one by encoding it with the
+-- appropriate headers.
+-- We would like to use Wire.Error.HttpError from wire-subsystems,
+-- but hscim can't depend on that.
+scimToWaiError :: ScimError -> (Wai.Error, [HTTP.Header])
+scimToWaiError err = (Wai.mkError e "scim-error" (LText.decodeUtf8 $ encode err), hs)
```

Comment:
this is stashing `err` away in the wai error description.  we should do the same thing as in `scimToServantError` and set status, label, ... from `err`.